docs/source/usage/tutorials/Deep/Dueling_DQN.rst

```diff
@@ -0,0 +1,101 @@
+======================
+Dueling Deep Q-Network
+======================
+
+Objective
+=========
+
+The main objective of DQN is to learn a function approximator for the Q-function using a 
```

Comment:
Again we should start of why we need a Dueling DQN i.e., the motivation behind it.